test/contact_properties.js

```diff
@@ -1,164 +1,296 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: process.env.API_KEY || 'demo' })
-
-const property = {
-  name: 'mk_customer_fit_segment',
-  label: 'MadKudu Customer Fit',
-  groupName: 'contactinformation',
-  description: 'MadKudu Customer Fit',
-  type: 'string',
-  fieldType: 'text',
-  formField: false,
-  displayOrder: -1,
-  options: [],
-}
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
 
 describe('contacts.properties', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
+  const properties = {
+    name: 'mk_customer_fit_segment',
+    label: 'MadKudu Customer Fit',
+    groupName: 'contactinformation',
+    description: 'MadKudu Customer Fit',
+    type: 'string',
+    fieldType: 'text',
+    formField: false,
+    displayOrder: -1,
+    options: [],
+  }
+  const deleteTestGroup = name => hubspot.contacts.properties.deleteGroup(name)
+  const createTestGroup = properties =>
+    hubspot.contacts.properties.createGroup(properties)
+
+  const deleteTestProperty = name => hubspot.contacts.properties.delete(name)
+  const createTestProperty = properties =>
```

Comment:
small naming suggestion: `createTestContactProperty`. Just to clarify this is a `ContactProperty` vs a `CompanyProperty`.